### PR TITLE
Roppenheimer patch 1

### PR DESCRIPTION
**Pull request by roppenheimer**
Section added to explain prediction lines, also some typos corrected.

### CODE REVIEW

**Comment by AdrianLxM** [comment]
Hi, @roppenheimer I believe part of the text about the prediction lines is already AAPS 2.0? ZT is not in 0.58.
IMHO it would be good in general to do the screenshots with current `dev` version as the other ones will be outdated soon. Just for comparison: Current `dev` does have more than twice the amount of commits than current `master`. @longboataline wrote some text in gitter yesterday (the one you copied it to FB). It would be great if you could combine that. I really like how you describe the GUI.

About the sitemap: It is a new file so won't cause merge conflicts. @typ1-brods is already reorganizing the whole structure of the wiki. On read the docs you can get an idea how it would/could look like if on the side panel in the lowes like you switch from "v:latest" to "v:dev".
Only the menu "English" is somehow finished, still work in progress, but that you get an impression :)

**Comment by roppenheimer** [comment]
Hi Adrian,

 

Thank you for your email. I am having difficulty understanding some of what you are saying. For example, what is “ZT”?

 

I do understand that the screenshots will need to be updated when version 2.0 becomes available – but for most people at this stage that is not the case and they are asking for guidance now. 

 

I find the structure of the project documentation on GitHub extremely confusing, and I know other people do too. It may be a suitable tool for developers, but that is not the audience that is now wanting to use the app. Maybe things were different at the early stages of development – but that stage has been passed and the audience has changed. There is a need for much more user friendly documentation, partly because it will help reduce the amount of support queries and also because people will judge the quality of the app by the documentation because that is the bit they can see.  I have had a look at the dev version of the wiki and I do not think that it is adequate either. It does not take the reader through a journey of how to understand AAPS, why they might want it, what it will do for them, how it works, how to build a rig, how to tune it and how to address the problems they may encounter.

 

I have met up with, and chatted with a fair number of people who are starting on this journey, they all have the same complaint: that although they see that this is something that could benefit them they are struggling to understand and engage with it. They get frustrated when they get told “read the wiki” – normally they have already read the wiki and they can’t understand it. They say it makes things more complicated than they need to be. I can see their point of view.

 

Sooner or later a wider audience of doctors will come into contact with AAPS. In my experience doctors are not the most patient of people, nor the most IT literate or innovative. If they do not engage easily and quickly with the app and the project they will discourage people from trying it and they will give the project a bad name. And that would be a huge shame.

 

My reason for getting involved was because I spent many hours reading everything that I could find in wikis, blogs, whatever. What I found was hard to follow, geeky, and full of errors and omissions. I even enlisted the help of a software developer who works for me to help and even he had difficulty following it.  Eventually I concluded that the only way to make sense of the documentation was to write my own.

 

I have been looking for an explanation of how the algorithm works. The only one I can find that makes much sense is the one written by Katie DiSimone and I understand that the Loop algorithm is somewhat different to the OpenAPS algorithm. This is important – I was at a meeting last week when someone said “I want to know how it works, I don’t want to plug a black box into myself and not know what it is doing”. Again, I can see her point of view.

 

I think there needs to be a re-appraisal of what, exactly, the documentation is supposed to be doing. As I said before, the project will be judged on it and as things stand it does not do the project any credit. I chatted last week with Katie DiSimone, she was of much the same view – the documentation she created for Loop is far more understandable and easy to navigate. In my view, and of a number of the other people I have spoken to, that is the kind of style and structure that AAPS documentation needs to have.

 

There are many ways in which people can, and do, contribute to the success of the OpenAPS/AAPS project: organising meetings, talking to their clinics, helping others find kit and consumables and supporting them to find ways around problems. Very few people will be getting involved in the actual development – and that is a good thing – or there is a danger of things becoming uncontrolled. When encouraging people to contribute to the project they should be encouraged to get involved in these other ways, not to make pull requests – that just intimidates them and makes them feel inadequate.

 

I’d sorry if this seems a bit blunt, but it urgently needs to be said. I do understand that a lot has been achieved by dedicated people in their spare time, but in a sense that frightens people from making criticisms and in the long term that could be disastrous for the project as a whole.

 

Best wishes.

 

Richard

**Comment by AdrianLxM** [comment]
Merge conflicts with current version. Probably outdated?
Please re-open if still valid.
